tests/Api/Responses/Expected/shop/updated_payment_method_on_order_response.json

```diff
@@ -37,8 +37,7 @@
             "variant": "\/api\/v2\/shop\/product-variants\/MUG_BLUE",
             "productName": "Mug",
             "id": @integer@,
-            "quantity": 3,
-            "originalPrice": null
```

Comment:
TBH I'm not sure about that field. I checked the master demo API endpoint
`api/v2/shop/account/orders/{tokenValue}/payments/{paymentId}`
 and it seems that that field should not be included, but I had one and it broke the tests